content/en/docs/tasks/administer-cluster/controller-manager-leader-migration.md

```diff
@@ -0,0 +1,109 @@
+---
+reviewers:
+- jpbetz
+- cheftako
+title: "Controller Manager Leader Migration"
+linkTitle: "Controller Manager Leader Migration"
+content_type: task
+---
+
+
+<!-- overview -->
+
+{{< feature-state state="alpha" for_k8s_version="v1.21" >}}
+
+{{< glossary_definition term_id="cloud-controller-manager" length="all" prepend="The cloud-controller-manager is">}}
+
+## Background
+
+As part of the cloud provider extraction effort, all cloud specific controllers must be moved out of the `kube-controller-manager`. All existing clusters that run cloud controllers in the `kube-controller-manager` must migrate to instead run the controllers in a cloud provider specific `cloud-controller-manager`.
```

Comment:
> cloud provider extraction

Can we link to more details about this, here / later in the page?

Reply:
Added a link to https://kubernetes.io/blog/2019/04/17/the-future-of-cloud-providers-in-kubernetes/